test/release/examples/primers/modules.chpl

```diff
@@ -0,0 +1,390 @@
+     outside that context.  There are several strategies for accomplishing this:
+
+     First, if only one symbol defined in the module is desired, and it is
+     desired only once, the symbol can be referenced explicitly - this is done
+     using the module name and a separating '.' as a prefix to the name of the
+     symbol desired.
+  */
+  var thriceFoo = 3 * modToUse.foo; // should be 36
+  writeln(thriceFoo);
+
+
+  {
+    /* If many of the module's symbols are desired, or the same symbol is
+       desired multiple times, then it is best to utilize what is known as a
+       "use statement".
+    */
+    use modToUse;
+
+    /*
+      Use statements can be inserted at the global scope, as well as any
+      arbitrary inner scope defined.
+    */
+
+    /* A use statement brings all of the visible symbols of a module into the
+       scope which defines the use statement.  These symbols may then be
+       accessed without the module name prefix.
+    */
+    var bazBarFoo = baz(bar, foo); // should be 28
+    writeln(bazBarFoo);
+  }
+
+
+
+  /* Once the scope defining a use statement is left, however, the symbols must
+     be accessed with a prefix.  The following line would cause an error,
+     because the previous use of the module 'modToUse' occurred within the curly
+     braces above:
+  */
+  // var twiceFoo = 2 * foo;
+
+
+
+  {
+    var bazBarFoo = baz(bar, foo); // should be 28
+
+    /* Use statements apply to the entire scope in which they are defined.  Even
+       if the use statement occurs after code which would access the symbols
+       without prefix, these accesses are still valid.
+    */
+    use modToUse;
+
+    writeln(bazBarFoo);
+  }
+
+
+  {
+    /* The symbols provided by a use statement are considered at an outer scope
+       to the symbols defined at the same level as the use itself.  Thus, if
+       another bar were defined here, and an access was attempted, the compiler
+       would find the bar at this scope, rather than modToUse.bar.
+    */
+    var bar = 4.0;
+
+    use modToUse;
+
+    writeln(bar); // Will output 4.0, not 2
+  }
+
+
+  var bar = false;
+  {
+    /* The symbols provided by a use statement are considered at an inner scope
+       to the symbols defined outside of the scope where the use statement
+       applies.  Thus, if another bar were defined outside of these curly
+       braces, and an access was attempted, the compiler would find the bar from
+       modToUse, rather than the outer bar.
+    */
+
+    use modToUse;
+    writeln(bar); // Will output 2, not false
+  }
+
+
+  {
+    /* Multiple modules may be used in the same use statement */
+    use modToUse, AnotherModule, ThirdModule;
+
+    if (a || b < 0) {
+      writeln(foo);
+    } else {
+      writeln(bar);
+    } // Will output foo (12)
+  }
+
+
+  {
+    /* And a scope may contain multiple use statements */
+    use modToUse;
+    use AnotherModule;
```

Comment:
These two examples are nice.  I  might add ", ThirdModule" to the second 'use' statement to drive home that multiple modules and multiple use statements are completely orthogonal.  You could also say "Equivalently, a scope may contain..." since there isn't any semantic difference between one use statement with multiple modules or multiple use statements (right?).  Or that could be made in a separate more explicit (and probably clear) statement.